molecule/zabbix_server/prepare.yml

```diff
@@ -73,6 +73,7 @@
       until: installation_dependencies is succeeded
       when:
```

Comment:
I think a better when condition would be `ansible_facts['os_family'] == 'Debian'`